test/logoutCallback.js

```diff
@@ -48,43 +48,6 @@ describe('GET /logout/:serviceProvider/callback', function onDescribe() {
     router.handle(req, res);
   });
 
-  it('should 401 when state incorrect', function onIt(done) {
-    const router = createRouter(mockExpress, correctConfig);
-
-    let redirectUrl = false;
-    const req = reqres.req({
-      url: '/auth/logout/callback/aprofiel',
-      method: 'GET',
-      query: {
-        state: '12345'
-      },
-      get: () => host,
-      session: {
-        save: (cb) => cb(),
-        user: {},
-        userToken: {},
-        aprofiel_logoutKey: '1234',
-        regenerate: (cb) => cb()
-      },
-    });
-    const res = reqres.res({
-      header: () => {},
-      redirect(val) {
-        redirectUrl = val
-        this.emit('end');
-      }
-    });
-
-    res.redirect.bind(res);
-
-    res.on('end', () => {
-      assert(res.sendStatus.calledWith(401));
-      return done();
-    });
-
-    router.handle(req, res);
-  });
-
   it('should 401 when serviceprovider unknown', function onIt(done) {
```

Comment:
Your test name doesn't align with your test

Reply:
Fixed. Will push it after the other remarks are resolved.